packages/devtools/lib/src/debugger/debugger.css

```diff
@@ -42,6 +48,29 @@
 
 .debugger-items-list {
     overflow-y: scroll;
+    cursor: pointer;
+}
+
+.open-popup {
+    z-index: 1000;
+    position: absolute;
+    top: 0px;
+    left: 5px;
+    height: 200px;
```

Comment:
maybe make this some % of the page instead of 200px?

Reply:
Yep using calc now.